main.py

```diff
@@ -24,7 +24,6 @@
 from datasets import build_dataset
 from engine import train_one_epoch, evaluate
 from samplers import RASampler
-import models
```

Comment:
This import is necessary to register the newly added models to `timm`, otherwise those models will not be available